docs/src/layouts/MainLayout.vue

```diff
@@ -161,4 +176,75 @@ $adjust-header-viewport: 860px;
   z-index: 5; // ensures it's lower than header but higher than components page filter section
 }
 
+@function generateRandomStars($number-of-stars) {
+  $value: '#{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  @for $i from 1 through $number-of-stars {
+    $value: '#{$value}, #{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  }
+  @return unquote($value);
+}
+
+$shadows-small: generateRandomStars(900);
+$shadows-medium: generateRandomStars(600);
+$shadows-big: generateRandomStars(500);
+
+#small-stars {
```

Comment:
You can problably use SCSS placeholders (eg. `%star` and `#star-shadow`) to group common CSS rules